packages/js/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nrwl/js",
   "version": "0.0.1",
-  "description": "JavaScript Plugin for Nx",
+  "description": "The Nx Plugin for Javascript contains executors and generators allowing your workspace to use next generation technologies/patterns and architectures.",
```

Comment:
I think executors and generators are implied for most plugins, can probably leave it out. Some wording issues too, but maybe something like this will work?


```
The Nx plugin that enables next generation technologies for JavaScript and TypeScript projects.
```

OR

```
The Nx plugin that provides the best experience for developing JavaScript and TypeScript projects.
```